src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,120 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ARRAYCOPY_POSITION = 0;
+    private static final int ARRAYCOPY_MOVE_POSITION = 1;
+    private static final double GROW_SIZE = 1.5;
+    private static final String INDEX_EXCEPTION_MESSAGE = "ArrayList index out of bounds";
+    private static final String NO_SUCH_ELEMENT_EXCEPTION_MESSAGE = "No such element";
```

Comment:
It's better not to keep exception messages in constants because message should contain index, size or element (should be changed dynamically)
Better use messages like: "Index: " + index + " is out of size: " + size
"Array list doesn't contain element: " + element
```suggestion
```